controller/Publish.php

```diff
@@ -34,26 +36,26 @@
  * @license GPL-2.0
  *
  */
-class Publish extends \tao_actions_CommonModule {
+class Publish extends \tao_actions_SaSModule {
 
     use OntologyAwareTrait;
     
-    public function wizard()
+    public function wizard(): void
     {
             try {
             $formContainer = new WizardForm(array('test' => $this->getRequestParameter('id')));
-            $myForm = $formContainer->getForm();
+            $form = $formContainer->getForm();
              
-            if ($myForm->isValid() && $myForm->isSubmited()) {
-                $test = $this->getResource($myForm->getValue('test'));
-                $env = $this->getResource($myForm->getValue('environment'));
+            if ($form->isValid() && $form->isSubmited()) {
```

Comment:
To avoid Cyclomatic complexity, I strongly suggest you refactor this code to avoid else conditions.